app/app/local_settings.py.dist

```diff
@@ -51,13 +51,14 @@ STATIC_URL = '/static/'
 
 # SECURITY WARNING: keep the secret key used in production secret!
 # required
-SECRET_KEY = 'TODO'
+SECRET_KEY = 'thisisasecret'
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.postgresql',
-        'NAME': 'postgres',
-        'USER': 'postgres',
-        'HOST': 'db',
+        'NAME': 'gitcoin',
+        'USER': 'gitcoin_user',
```

Comment:
Any reason you updated these?  They are set to the defaults on the docker image for PSQL at the moment and this will probably break everyone else's local environments.

Reply:
Oops didn't mean to upload these changes.

Reply:
Also the README says to create this user for local development.